packages/webapp/src/inductions/components/induction-journeys/common/induction-journey-container.tsx

```diff
@@ -1,123 +1,147 @@
+    CreateInvite = "inviter-create-invite",
+    PendingProfile = "inviter-pending-profile",
+    VideoAndEndorse = "inviter-video-and-endorse",
+    PendingDonation = "inviter-pending-donation",
+    Complete = "inviter-complete",
+}
+
 export const INVITER_INDUCTION_STEPS: Step[] = [
     {
+        key: InductionStepInviter.CreateInvite,
         title: "CREATE INVITE",
         text: "Add invitee and witnesses by EOS account.",
     },
     {
+        key: InductionStepInviter.PendingProfile,
         title: "INVITEE PROFILE",
         text: "Invitee must log in and set up their profile.",
     },
     {
+        key: InductionStepInviter.VideoAndEndorse,
         title: "INDUCT & ENDORSE",
         text:
             "Record and attach induction ceremony. Inviter and witnesses endorse invitee.",
     },
     {
+        key: InductionStepInviter.PendingDonation,
         title: "INVITEE DONATION",
         text: `Invitee donates ${assetToString(
             minimumDonationAmount
         )} to the Eden community.`,
     },
     {
+        key: InductionStepInviter.Complete,
         title: "ALL DONE",
         text: "NFTs are minted. We have a new member!",
     },
 ];
 
+export enum InductionStepGenesis {
+    Profile = "genesis-profile",
+    Donate = "genesis-donate",
+    StandBy = "genesis-standby",
+    Complete = "genesis-complete",
+}
+
 export const GENESIS_INDUCTION_STEPS: Step[] = [
     {
+        key: InductionStepGenesis.Profile,
         title: "SET UP YOUR PROFILE",
         text: "Let the community know who you are.",
     },
     {
+        key: InductionStepGenesis.Donate,
         title: "DONATE",
         text: `Give ${assetToString(
             minimumDonationAmount
         )} to the Eden community.`,
     },
     {
+        key: InductionStepGenesis.StandBy,
         title: "STAND BY",
         text:
             "All Genesis members must complete the process for the community to go live.",
     },
     {
+        key: InductionStepGenesis.Complete,
         title: "YOU'RE IN",
         text: "The community is activated. Welcome to Eden.",
     },
 ];
 
-export enum InductionRole {
-    INVITEE = "invitee",
-    INVITER = "inviter",
-    GENESIS = "genesis",
-}
-
 interface Props {
-    role: InductionRole;
-    step: 1 | 2 | 3 | 4 | 5;
-    vAlign?: "top" | "center";
+    step: InductionStep;
     children: React.ReactNode;
 }
 
-export const InductionJourneyContainer = ({
-    role,
-    step,
-    vAlign = "center",
-    children,
-}: Props) => {
-    let steps: Step[];
-
-    switch (role) {
-        case InductionRole.GENESIS:
-            steps = GENESIS_INDUCTION_STEPS;
-            break;
-        case InductionRole.INVITEE:
-            steps = INVITEE_INDUCTION_STEPS;
-            break;
-        case InductionRole.INVITER:
-            steps = INVITER_INDUCTION_STEPS;
-            break;
+export const InductionJourneyContainer = ({ step, children }: Props) => {
+    function isStepIn<T>(steps: T) {
```

Comment:
🏹 arrow function? :)

Reply:
Sure! I'm still fuzzy on it... is there a benefit one way or the other in cases like this?

Reply:
not really, just keeping the codebase consistent